roles/login_ocp/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: debug info
+  debug:
+   msg: "Logging into cluster using KUBECONFIG: {{ ansible_env.KUBECONFIG }}"
```

Comment:
This is good, I would simply mention in the Readme that the user must have exported the `KUBECONFIG` env var prior to this.

What feels weird to me is that you are logging in with user `admin` and using the `kubeadmin_password` which I'm assuming comes from the `openshift-install` binary right? Shouldn't the user be `kubeadmin`?

Reply:
I decided to keep it that way because, this playbooks are not in the same dir where cluster was deployed. We can also ask user to export password as env variable.

If you deploy cluster with https://github.com/fusor/origin3-dev, user is going to be ```admin```.

Reply:
I think either rename the file to `admin_password` just for consistency or use an env var. Only reason I load it from a file in the other repo is that the `openshift-install` binary places the password on disk already. Feels weird to expect the user to put their password in a file.

Reply:
I changed the playbook, now it uses env variable